src/buza/templates/buza/subject_list.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'base.html' %}
-{% load static %}
+{% load static buza_tags%}
```

Comment:
I tend to try to keep template tag imports on separate lines, for easier management.